homeassistant/components/hardkernel/manifest.json

```diff
@@ -7,5 +7,6 @@
   "dependencies": ["hardware"],
   "documentation": "https://www.home-assistant.io/integrations/hardkernel",
   "integration_type": "hardware",
+  "quality_scale": "legacy",
```

Comment:
Is this expected for `hardware` integrations?

Reply:
I knew there was one core file in there but couldn't figure out which one it was. I am now taking off ✈️, so if you spot more feel free to update :) (I couldn't find others after going through the list twice)